hydromt_wflow/workflows/landuse.py

```diff
@@ -486,7 +486,7 @@ def add_planted_forest_to_landuse(
         GeoDataFrame containing planted forest data. Required columns are: 'geometry',
         and optionally 'forest_type' to find orchards.
     ds_like : xr.Dataset
-        Dataset at model resolution. Required variables are 'USLE_C'.
+        Dataset at model resolution. Required variables are 'erosion_usle_c'.
```

Comment:
Here in a workflow I prefer to use hydromt convetion name ie `usle_c`. Can you update the rest?